iceoryx_utils/test/moduletests/test_cxx_string.cpp

```diff
@@ -361,6 +361,23 @@ TEST(String100, UnsafeCharToStringConstrIncludingNullCharWithCountResultsInSizeC
     EXPECT_THAT(testSubject.c_str(), StrEq("ice\0ryx"));
 }
 
+TEST(CharArrayAssignment, AssignZeroTerminatedCharArrayOfSizeForFullCapa)
```

Comment:
The test name confuses me, but ok. OfSize? SizeCapacity? How about AssignZeroTerminatedCharArrayOfSizeFullCapacity. Does not really matter though, those names are bound to be strange.

Reply:
Yeah, basically it's not telling what the size is, CharArrayOfSize???
I also prefer capacity.

Reply:
how about AssignZeroTerminatedCharArray**With**SizeForFullCapa?

Reply:
Better

Reply:
Fine with me.

Reply:
maybe @FerdinandSpitzschnueffler has an even better idea. This could probably be renamed in #253